tests/csm/rest/test_capacity_usage.py

```diff
@@ -74,12 +74,12 @@ def setup_class(cls):
         cls.restore_method = None
         cls.restore_list = []
         cls.deployment_name = []
-        cls.failed_pod = []
+        cls.failed_pod = cls.failed_pod_restore = []
         cls.deployment_backup = None
         cls.restore_pod_data = None
         cls.set_name = None
         cls.num_replica = 0
-        cls.deploy_list = cls.master.get_deployment_name(POD_NAME_PREFIX)
+        cls.deploy_list = cls.master.get_all_pods_and_ips("data").keys() #cls.master.get_deployment_name(POD_NAME_PREFIX)
```

Comment:
can we remove #cls.master.get_deployment_name(POD_NAME_PREFIX) if not required ?

Reply:
removed